hw/ip/edn/data/edn.prj.hjson

```diff
@@ -7,8 +7,8 @@
     design_spec:        "../doc",
     dv_doc:             "../doc/dv",
     hw_checklist:       "../doc/checklist",
-    version:            "0.5",
+    version:            "0.9",
```

Comment:
According to https://docs.opentitan.org/doc/project/development_stages/#versioning the version is orthogonal to the L/V/D/S states and indicates that something in the spec changed. That's not the case here, so please leave the version number as-is.

Reply:
After reading this section (had not before), one could interpret it that way. There seems to be a theme of known design requirements up front, in which case you might set the version to 1.0. I took the route of this design is evolving (of which it has) so set the level originally at 0.5. Now that requirements, topology, etc. has converged, I thought is would be appropriate to up the version to 0.9, probably to "catch-up" with the specification level. I can certainly leave it at 0.5, but I do not think that is particularly helpful in conveying the design version.

Reply:
We do have a fair amount of IP blocks in the tree at the moment at very different versions. Currently, the version doesn't have any semantics -- we just chose a random number and went with it. We can (and likely should) align them at one point, but let's do so in a project-wide cleanup, and not as part of this PR please.

Reply:
Fine with me, was trying to make this version number possibly more meaningful, but if a global scrubbed is planned, I will leave it as is.